spec/facades/food_facade_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe 'FoodFacade' do
+  it "returns an array of food objects", :vcr do
+
+    facade = FoodFacade.service("sweet potatoes")
+
+    expect(facade).to be_an Array
+    expect(facade.first).to be_a Hash
```

Comment:
These expectations could be more robust. Would have also liked to see the `food_info` method unit tested.